proposals/NNNN-unlock-existential-types-for-all-protocols.md

```diff
@@ -120,17 +120,17 @@ extension Animal {
 }
 ```
 
-Contrary to the intent, `Animal` is still assumed as "having an associated type requirement", which happens to be the only thing restraining the existential.
+Unfortunately, `Animal` is still "assumed" to have an associated type requirement, which is solely responsible for restraining the existential.
```

Comment:
* Why use "unfortunately" where we can afford to state why something is unfortunate?
* "assumed as having" means "treated as having", "assumed to have" is a conjecture.

Reply:
`"assumed as having" means "treated as having", "assumed to have" is a conjecture.`

Sorry for misinterpreting that.

`Why use a sighing "unfortunately" where we can afford to state why something is unfortunate?`

I don't mind using "Contrary to the intent" so long as we specify whose intent –– otherwise it is, in my opinion, better to use a well-known, widely used connector. I assume you mean: Contrary to the author's intent.

  ```suggestion
Contrary to the author's intent, `Animal` is still assumed as having an associated type requirement, which is solely responsible for restraining the existential.
```

Reply:
Agreed. Let us use "user" or "developer" though, otherwise it might seem as if we are referring to the authors of the proposal. Feel free to edit and commit the suggestion.